app/controllers/coronavirus_form/expert_advice_controller.rb

```diff
@@ -5,46 +5,35 @@ class CoronavirusForm::ExpertAdviceController < ApplicationController
   include FieldValidationHelper
 
   def show
-    session[:expert_advice] ||= []
```

Comment:
Is there not an advantage in memosing something here?

Reply:
We're now storing a Yes/No value instead of an array, so there's no need to set a default value anymore